src/App.tsx

```diff
@@ -7,6 +7,7 @@ import { Todos } from './interfaces';
 
 const App = () => {
   const [todos, setTodos] = useState<Todos[]>([]);
+  const [initialTodos, setInitialTodos] = useState<Todos[]>([]);
```

Comment:
do sorting on rendering, it's too expensive to store two lists if they are very big